### PR TITLE
Add userReports query

### PR DESCRIPTION
**Pull request by queq1890**
# Story URL 
https://trello.com/c/D6IjJqzA/14-3-sprint2-daily-report-listing-page

# What I did 
- adding `userReports` query, which returns all daily reports by signed_in user

### CODE REVIEW

File: src/schema.graphql [hunk 1]
```diff
@@ -9,24 +9,24 @@ input IssueInput {
...
-  info: String!,
+  info: String!
   divisions: [Division!]!
   division(where: DivisionWhereUniqueInput!): Division
   dailyReports: [DailyReport!]!
   dailyReport(where: DailyReportWhereUniqueInput!): DailyReport
   issues: [Issue!]!
   issue(where: IssueWhereUniqueInput!): Issue
   me: User!
+  currentUserReports: [DailyReport!]!
```
Comment (queq1890) [inline src/schema.graphql]: If you have better naming for this query, let me know 🙇‍♂️

**Review by giathinh910 — APPROVED** [review]

File: src/schema.graphql (hunk 1 above)
Comment (phuongthuan) [reply src/schema.graphql]: @queq1890 The naming is should be only `userReports` 
Why ? 
The purpose is reuse query.
If I'm the current user, I can easily get other users by this query that don't need create a new query.

**Review by phuongthuan — APPROVED** [review]

File: src/schema.graphql (hunk 1 above)
Comment (queq1890) [reply src/schema.graphql]: @phuongthuan I'm gonna rename it. Thx!

**Comment by queq1890** [comment]
Merging ~~

**Pull request merged**